std/builtin/base.neva

```diff
@@ -59,5 +59,13 @@ pub component Unwrap<T any>(data maybe<T>) (some T, none struct{})
 #extern(list list_len, map map_len)
 pub component Len<T list<any> | map<any>>(data T) (res int)
 
+#extern(list_push)
+pub component Push<T> (lst list<T>,data T) (res list<T>)
+
+
+#extern(list_sort)
+pub component Sort<T> (data list<T>) (res T)
```

Comment:
I took inspiration from JS and it actually converts any non number to a string and sorts them alphabetically, would you prefer it to be overloaded? This seems unnecessary since you have to implement one for each type

Reply:
> I took inspiration from JS

That's not perfectly designed language :D

IMO that sounds unintuitive. I wouldn't assume any implicit conversions under the hood, also that takes some time which is bad for performance.

It would be cool to see here a union constraint `T int | float | string` and no implicit conversions under the hood.

> you prefer it to be overloaded?

Personally yes, I see no problem in copy-pasting 50 lines of code couple of times. You'll have clear separate implementations. Otherwise you'll have to type-check what type message have to figure out how to retrieve data (call .Int() or .String() or .Float()).

That's just one cheap call tho so I wont enforce overloading here. But union constraint and no implicit conversions is a requirement